### DISCUSSION TITLE
Open links in new tab by default

### DISCUSSION THREAD

**Discussion by InfectedIsm** [solodit/feedback#42] [New Feature Requests]
Hi guys, 

would be pretty interesting imo to have links to open in a new tab, rather than in the actual tab.
The reason is, right now Solodit does not save the current search. 
So when opening a link (if we forget to ctrl+click for a new tab), if we go back to previous page, we're sent back to the default dashboard which is pretty inconvenient.

And thanks a lot for your work, its never said enough but your tool is probably one of the most important of the web3 security environment.

Cheers

**Comment by wang1252** [comment]
Fixed this problem. Thank you for your suggestion.
